src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -669,6 +669,11 @@ private static CultureData CreateCultureWithInvariantData()
                 return CultureData.Invariant;
             }
 
+            if (GlobalizationMode.Invariant && GlobalizationMode.PredefinedCulturesOnly)
```

Comment:
Why not move it to the check below and remove the invariant assert there?

Reply:
done moving it.